### PR TITLE
OHRM5X-1105: Developed Employee Trackers List View

### PR DESCRIPTION
**Pull request by ChenukaGamage**

### CODE REVIEW

**Review by Super-Chama — CHANGES_REQUESTED** [review]

File: symfony/client/src/orangehrmPerformancePlugin/pages/EmployeeTrackers.vue [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+import {computed, ref} from 'vue';
+import EmployeeAutocomplete from '@/core/components/inputs/EmployeeAutocomplete';
+import {APIService} from '@/core/util/services/api.service';
+import usePaginate from '@ohrm/core/util/composable/usePaginate';
+import useSort from '@ohrm/core/util/composable/useSort';
+
+const userdataNormalizer = data => {
+  return data.map(item => {
+    return {
+      empName: item.empName,
```
[the remark this answers is not in the document]
Comment (ChenukaGamage) [reply symfony/client/src/orangehrmPerformancePlugin/pages/EmployeeTrackers.vue:101]: I removed the normaliser.

File: symfony/client/src/orangehrmPerformancePlugin/pages/EmployeeTrackers.vue [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+</template>
+
+<script>
+import {computed, ref} from 'vue';
+import EmployeeAutocomplete from '@/core/components/inputs/EmployeeAutocomplete';
+import {APIService} from '@/core/util/services/api.service';
+import usePaginate from '@ohrm/core/util/composable/usePaginate';
+import useSort from '@ohrm/core/util/composable/useSort';
+
+const userdataNormalizer = data => {
```
[the remark this answers is not in the document]
Comment (ChenukaGamage) [reply symfony/client/src/orangehrmPerformancePlugin/pages/EmployeeTrackers.vue:98]: I have removed it now.

File: symfony/client/src/orangehrmPerformancePlugin/pages/EmployeeTrackers.vue [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+      await this.execQuery();
+    },
+    async filterItems() {
+      await this.execQuery();
+    },
+  },
+};
+</script>
+
+<style scoped></style>
```
[the remark this answers is not in the document]
Comment (ChenukaGamage) [reply symfony/client/src/orangehrmPerformancePlugin/pages/EmployeeTrackers.vue:248]: Updated

File: symfony/client/src/orangehrmPerformancePlugin/pages/EmployeeTrackers.vue [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+            },
+          },
+        },
+      ],
+      includeOpts: [
+        {id: 1, param: 'onlyCurrent', label: 'Current Employees Only'},
+        {id: 2, param: 'currentAndPast', label: 'Current and Past Employees'},
+        {id: 3, param: 'onlyPast', label: 'Past Employees Only'},
+      ],
+      checkedItems: [],
```
[the remark this answers is not in the document]
Comment (ChenukaGamage) [reply symfony/client/src/orangehrmPerformancePlugin/pages/EmployeeTrackers.vue:232]: Updated

File: symfony/client/src/orangehrmPerformancePlugin/pages/EmployeeTrackers.vue [hunk 5]
```diff
@@ -0,0 +1,248 @@
...
+    };
+  },
+
+  methods: {
+    async resetDataTable() {
+      this.checkedItems = [];
```
[the remark this answers is not in the document]
Comment (ChenukaGamage) [reply symfony/client/src/orangehrmPerformancePlugin/pages/EmployeeTrackers.vue:238]: Updated

File: symfony/client/src/orangehrmPerformancePlugin/pages/EmployeeTrackers.vue [hunk 6]
```diff
@@ -0,0 +1,248 @@
...
+    <br />
+    <div class="orangehrm-paper-container">
+      <table-header
+        :selected="checkedItems.length"
+        :total="total"
+        :loading="isLoading"
+      ></table-header>
+      <div class="orangehrm-container">
+        <oxd-card-table
+          v-model:selected="checkedItems"
```
[the remark this answers is not in the document]
Comment (ChenukaGamage) [reply symfony/client/src/orangehrmPerformancePlugin/pages/EmployeeTrackers.vue:69]: Updated

File: symfony/plugins/orangehrmPerformancePlugin/config/routes.yaml [hunk 7]
```diff
@@ -1,4 +1,9 @@
-performance_module_under_development: # remove this
-  path: /performance/{screen}
-  controller: OrangeHRM\Core\Controller\Common\ModuleUnderDevelopmentController::handle
+#performance_module_under_development: # remove this
```
[the remark this answers is not in the document]
Comment (ChenukaGamage) [reply symfony/plugins/orangehrmPerformancePlugin/config/routes.yaml:1]: Removed

File: symfony/plugins/orangehrmPerformancePlugin/config/routes.yaml [hunk 8]
```diff
@@ -1,4 +1,9 @@
...
+#  path: /performance/{screen}
+#  controller: OrangeHRM\Core\Controller\Common\ModuleUnderDevelopmentController::handle
+#  methods:
+
+performance_employee_tracker: # remove this
```
[the remark this answers is not in the document]
Comment (ChenukaGamage) [reply symfony/plugins/orangehrmPerformancePlugin/config/routes.yaml:6]: Removed

**Review by Super-Chama — APPROVED** [review]

**Review by RajithaKumara — APPROVED** [review]

**Review by SharukaNP — APPROVED** [review]

**Pull request merged**